### PR TITLE
QPID-8473:[Broker-J]Added operational logs for sender links

### PR DESCRIPTION
**Pull request by Dedeepya-T**
Added operational logs for sender links attaching to queue and exchange. 
This is implemented specifically for clients using AMQP 1.0 protocol only.

### CODE REVIEW

File: broker-core/src/main/java/org/apache/qpid/server/logging/messages/Sender_logmessages.properties [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#
+CREATE = SND-1001 : Create : {0} : {1}
+CLOSE = SND-1002 : Close : {0} : {1}
```
Comment (alex-rufous) [inline broker-core/src/main/java/org/apache/qpid/server/logging/messages/Sender_logmessages.properties:21]: Please add new line

File: broker-core/src/main/java/org/apache/qpid/server/model/PublishingLink.java [hunk 2]
```diff
@@ -23,6 +23,7 @@
 @ManagedAttributeValueType(isAbstract = true)
 public interface PublishingLink extends ManagedAttributeValue
 {
+    String TYPE = "link";
```
Comment (alex-rufous) [inline broker-core/src/main/java/org/apache/qpid/server/model/PublishingLink.java:26]: Please rename constant TYPE into TYPE_LINK

**Review by alex-rufous — CHANGES_REQUESTED** [review]

File: broker-core/src/main/java/org/apache/qpid/server/model/PublishingLink.java (hunk 2 above)
Comment (Dedeepya-T) [reply broker-core/src/main/java/org/apache/qpid/server/model/PublishingLink.java]: Done!

File: broker-core/src/main/java/org/apache/qpid/server/logging/messages/Sender_logmessages.properties (hunk 1 above)
Comment (Dedeepya-T) [reply broker-core/src/main/java/org/apache/qpid/server/logging/messages/Sender_logmessages.properties]: Done!

**Review by alex-rufous — CHANGES_REQUESTED** [review]
Dedeepya,
The changes look good. Please merge them into a single commit

**Comment by Dedeepya-T** [comment]
> Dedeepya,
> The changes look good. Please merge them into a single commit

Done Alex!

**Pull request closed without merging**